backend/iot_mobile/src/main/kotlin/edu/pwr/iotmobile/rabbit/RabbitAdminConfig.kt

```diff
@@ -15,15 +15,15 @@ import org.springframework.context.annotation.Scope
 @Configuration
 class RabbitAdminConfig(
```

Comment:
I don't think passwords should be included in code, even if it's the default ones